snmp.yml

```diff
@@ -1565,3 +1565,132 @@ fortinet_fortigate:
           type: Integer32
         - labelname: fgWcWtpSessionWtpId
           type: OctetString
+
+# ServerTech Sentry 3 MIB
+#
+# Used by ServerTech PDUs
+#
+# ftp://ftp.servertech.com/Pub/SNMP/sentry3/Sentry3OIDTree.txt
+# ftp://ftp.servertech.com/Pub/SNMP/sentry3/Sentry3.mib
+servertech_sentry3:
+  version: 2
+  auth:
+    community: public
+  walk:
+    - 1.3.6.1.2.1.1
+    - 1.3.6.1.4.1.1718.3.2.2.1
+    - 1.3.6.1.4.1.1718.3.2.3.1
+  metrics:
+    - name: sysUpTime
+      oid: 1.3.6.1.2.1.1.3
+    # Infeed metrics
+    - name: infeed_load_hundredths_amper
+      oid: 1.3.6.1.4.1.1718.3.2.2.1.7
+      indexes:
+        - labelname: towerIndex
+          type: Integer32
+        - labelname: infeedIndex
+          type: Integer32
+      lookups:
+        - labels: [towerIndex, infeedIndex]
+          labelname: infeedName
+          oid: 1.3.6.1.4.1.1718.3.2.2.1.3
+    - name: infeed_votage_tenths_volt
+      oid: 1.3.6.1.4.1.1718.3.2.2.1.11
+      indexes:
+        - labelname: towerIndex
+          type: Integer32
+        - labelname: infeedIndex
+          type: Integer32
+      lookups:
+        - labels: [towerIndex, infeedIndex]
+          labelname: infeedName
+          oid: 1.3.6.1.4.1.1718.3.2.2.1.3
+    - name: infeed_watts
+      oid: 1.3.6.1.4.1.1718.3.2.2.1.12
+      indexes:
+        - labelname: towerIndex
+          type: Integer32
+        - labelname: infeedIndex
+          type: Integer32
+      lookups:
+        - labels: [towerIndex, infeedIndex]
+          labelname: infeedName
+          oid: 1.3.6.1.4.1.1718.3.2.2.1.3
+    - name: infeed_hundred_watt_hours
+      oid: 1.3.6.1.4.1.1718.3.2.2.1.16
+      indexes:
+        - labelname: towerIndex
+          type: Integer32
+        - labelname: infeedIndex
+          type: Integer32
+      lookups:
+        - labels: [towerIndex, infeedIndex]
+          labelname: infeedName
+          oid: 1.3.6.1.4.1.1718.3.2.2.1.3
+    # Outlet metrics
+    - name: outlet_load_hundredths_amper
+      oid: 1.3.6.1.4.1.1718.3.2.3.1.7
+      indexes:
+        - labelname: towerIndex
+          type: Integer32
+        - labelname: infeedIndex
+          type: Integer32
+        - labelname: outletIndex
+          type: Integer32
+      lookups:
+        - labels: [towerIndex, infeedIndex, outletIndex]
+          labelname: outletName
```

Comment:
So, by that logic, we should drop support for lookups, as lookups in SNMP are never guaranteed to be unique.  And there's never a case where we should allow mapping them into labels.